peac/cpp/plane_fitter_pcd.cpp

```diff
@@ -291,6 +292,7 @@ int process() {
 	} else {
 		std::string inputDir = global::getFileDir(filelist);
 
+		filelist = filelist.substr(0, filelist.size() - 1);
```

Comment:
2)On Linux platforms custom iniGet reads parameter with new line symbol at the end getting, I fixed it here